app/police-custody-form/controllers.js

```diff
@@ -23,6 +23,7 @@ exports.addEvents = async function (req, res) {
 
   await req.services.event.postEvents(lockoutEvents, move, journeys, user)
 
+  req.session.successBanner = true
```

Comment:
`req.flash` should store the message in the session, so you should be able to call it here directly before the redirect and avoid the logic in the timeline view. This also means we can change the redirect without having to change that logic, and keeps it close to where the success happens.

Reply:
Ohh nice!